server/models/trip.model.js

```diff
@@ -0,0 +1,14 @@
+import mongoose from 'mongoose';
+
+const tripSchema = new mongoose.Schema({
+  country: String,
```

Comment:
Regarding the validation it could be useful to set some keys to required, like e.g:
category: { type: String, required: true}

But I'm just talking big, I haven't validated my schemes either. But I will ;)

Reply:
thanks for checking my code. yes i know. Generally, Validation is not easy for me. But i will try... ;)